config/log.conf

```diff
@@ -7,7 +7,8 @@ log_format main_json escape=json
   '{'
     '"body_bytes_sent":"$body_bytes_sent",'
     '"host":"$host",'
-    '"http_referrer":"$http_referer",'
+    '"http_connection":"$http_connection",'
```

Comment:
Because it would be nice to see what the value of the `Connection` header is set to. Currently there's no way to see what it is. I also thought of the adding one for the protocol version but we can get that from the `request` attribute who's value includes the protocol i.e. `GET / HTTP/1.1` as shown below.

```json
{
  "body_bytes_sent": "1253",
  "host": "test.example.com",
  "http_referrer": "",
  "http_user_agent": "Mozilla/5.0 (apple-x86_64-darwin20.1.0) Siege/4.0.7",
  "http_x_forwarded_for": "105.163.39.239",
  "proxy_x_forwarded_port": "443",
  "proxy_x_forwarded_proto": "https",
  "proxy_x_forwarded_ssl": "on",
  "proxy_x_request_id": "eaec9ce4b52e915d59248fa2de3aa301",
  "remote_addr": "10.23.72.250",
  "remote_user": "",
  "request": "GET / HTTP/1.1",
  "request_length": "313",
  "request_time": "0.002",
  "status": "200",
  "time_iso8601": "2021-02-19T10:26:08+00:00"
}
```